Makefile

```diff
@@ -38,7 +38,7 @@ lint: lint-proto lint-go  ## Lint code and protos
 
 .PHONY: lint-go
 lint-go: $(BIN)/golangci-lint
-	$(BIN)/golangci-lint run ./...
+	$(BIN)/golangci-lint run --modules-download-mode=readonly --timeout=3m0s ./...
```

Comment:
Huh, this is interesting! @oliversun9 can you also update connectrpc/connect-go and bufbuild/skeleton-oss-go to use this flag? I think those are the repositories we usually use as templates, so if we update them the change will eventually spread to the other repos.

Reply:
Cool! They are here: https://github.com/connectrpc/connect-go/pull/617 and https://github.com/bufbuild/skeleton-oss-go/pull/8